### PR TITLE
Add `mc-sgx-core-build` crate

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
Add the `mc-sgx-core-build` crate to hold common build utilities used
with other SGX crates.

### CODE REVIEW

File: core/build/README.md
Comment (remoun) [inline core/build/README.md:10]: Stale references
```suggestion
```

File: trusted/trts/sys/Cargo.toml
Comment (remoun) [inline trusted/trts/sys/Cargo.toml:16]: Nit: Add a newline to maintain separate sorted groups
```suggestion

mc-sgx-core-build = { path = "../../../core/build" }
```

File: untrusted/test_enclave/Cargo.toml
Comment (remoun) [inline untrusted/test_enclave/Cargo.toml:20]: Nit: Add a newline to maintain separate sorted groups
```suggestion

mc-sgx-core-build = { path = "../../core/build" }
```

File: untrusted/urts/sys/Cargo.toml
Comment (remoun) [inline untrusted/urts/sys/Cargo.toml:17]: Nit: Add a newline to maintain separate sorted groups
```suggestion

mc-sgx-core-build = { path = "../../../core/build" }
```

File: untrusted/urts/sys/types/Cargo.toml
Comment (remoun) [inline untrusted/urts/sys/types/Cargo.toml:16]: Nit: Add newlines
```suggestion

mc-sgx-core-build = { path = "../../../../core/build" }

```

**Review by remoun — APPROVED** [review]

File: core/build/README.md
Comment (nick-mobilecoin) [reply core/build/README.md:10]: doh, copy pasta fail, thanks!

**Review by awygle — APPROVED** [review]
LGTM

**Comment by nick-mobilecoin** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/26?utm_source=stack-comment).


Job Reference: QB8STOU63qtQFHJyhfrd

**Pull request merged**
